README.md

```diff
@@ -1,91 +1,129 @@
+---
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
 # wtf-workshop
 
 <!-- badges: start -->
+
 <!-- badges: end -->
 
+# 2022 RStudio Conference Overivew
```

Comment:
I feel like this belongs *after* the section that explains the purpose of this repo and that gives an overview of where we keep different types of material.

I don't feel super strongly about this if you do, though.

Reply:
done